quickevent/quickevent.iss

```diff
@@ -22,99 +10,19 @@ AppPublisherURL=http://www.{#APP_NAME_LOWER}.cz
 AppSupportURL=http://www.{#APP_NAME_LOWER}.cz
 AppUpdatesURL=http://www.{#APP_NAME_LOWER}.cz
 DefaultDirName=C:\{#APP_NAME}
-DisableDirPage=no
 DefaultGroupName={#APP_NAME}
 OutputDir={#BUILD_DIR}\_inno\{#APP_NAME_LOWER}
-#ifdef INCLUDE_QT
-OutputBaseFilename={#APP_NAME_LOWER}-{#VERSION}-{#COMPANY}-setup
-#else
-OutputBaseFilename={#APP_NAME_LOWER}-{#VERSION}-{#COMPANY}-noqt-setup
-#endif
-;SetupIconFile=.\divers\quickevent\images\{#APP_NAME_LOWER}.ico
+DisableDirPage=no
+OutputBaseFilename={#APP_NAME_LOWER}-{#VERSION}-setup
 Compression=lzma
 SolidCompression=yes
 
-[Languages]
-;Name: english; MessagesFile: compiler:Default.isl
-;Name: czech; MessagesFile: compiler:Czech.isl
-
 [Tasks]
 Name: desktopicon; Description: {cm:CreateDesktopIcon}; GroupDescription: {cm:AdditionalIcons}; Flags: unchecked
 Name: quicklaunchicon; Description: {cm:CreateQuickLaunchIcon}; GroupDescription: {cm:AdditionalIcons}; Flags: unchecked
 
 [Files]
-Source: {#BUILD_DIR}\bin\{#APP_NAME_LOWER}.exe; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\quickshow.exe; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\quickhttpd.exe; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\qsqlmon.exe; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\qfcore.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\qfqmlwidgets.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\quickeventcore.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\quickeventgui.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\siut.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#BUILD_DIR}\bin\necrolog.dll; DestDir: {app}; Flags: ignoreversion
-
-Source: {#BUILD_DIR}\bin\qml\*; DestDir: {app}\qml; Flags: ignoreversion recursesubdirs createallsubdirs; Excludes: "*.debug"
-Source: {#SRC_DIR}\quickevent\app\quickevent\datafiles\*; DestDir: {app}\quickevent-data; Flags: ignoreversion recursesubdirs createallsubdirs
-
-; NOTE: Don't use "Flags: ignoreversion" on any shared system files
-
-Source: {#SSL_DIR}\libcrypto-1_1-x64.dll; DestDir: {app}; Flags: ignoreversion
-Source: {#SSL_DIR}\libssl-1_1-x64.dll; DestDir: {app}; Flags: ignoreversion
```

Comment:
apart of `libpq` (and its dependencies) the `libssl` and `libcrypto` also needs to be part of the installation. version 3.0 for Qt6 and 1.1 fot Qt5.

I might focus only on the Qt6 build at this point

Reply:
I don't plan on supporting qt5 with this. The project can't even build with it.

AFAIK, you don't need libssl/libcrypto on windows anymore, Qt6 can use built-in Windows implementation of SSL.